R/dsb.r

```diff
@@ -46,7 +46,12 @@
 DSBNormalizeProtein = function(cell_protein_matrix, empty_drop_matrix,
                                denoise.counts = TRUE,  use.isotype.control = TRUE, isotype.control.name.vec = NULL,
                                define.pseudocount = FALSE,pseudocount.use){
-
+  if(!is.null(isotype.control.name.vec)) {
+      if(!all(isotype.control.name.vec %in% rownames(cell_protein_matrix))) {
```

Comment:
Thanks for this! I'll merge this branch.

In my next version I will probably also add a print statement for 
`setdiff(isotype.control.name.vec, rownames(cell_protein_matrix))`
so the user sees where the discrepancy is. 

Thanks for contributing.